Resources/Prototypes/_Goobstation/Loadouts/loadout_groups.yml

```
@@ -1014,7 +1018,7 @@
   - BSOArmoredCoat # EE
   - BSOMarineVest # EE
   - BSOArmoredVest # EE
-  - BlueshieldOfficerOuterKimonoBlueshield
+  - BlueshieldOfficerOuterKimonoBlueshield #Omu
```

```suggestion
  - BlueshieldOfficerOuterKimonoBlueshield
```
Seems to be from Goob already